CAP/doc/AddFunctions.autodoc

```diff
@@ -138,8 +138,16 @@ The record can have the following components, used as described:
                             Please note that if you have a method selection argument for your function, you need to give the argument_list
                             to explicitly state which argument is the method selection argument.
 
-* return_type (optional): The return type can be object, morphism, or twocell. If it is one of those, the correct add function (see above) is
-                          used for the result of the computation. Otherwise, no add function is used after all.
+* return_type (optional): The return type can be one of the following:
+                          * <C>object</C>,
+                          * <C>morphism</C> or <C>morphism_or_fail</C>,
```

Comment:
It looks a bit weird to bundle this tbh . . .

Reply:
I bundled this because I think about this as "a function can return an object, a morphism (with the special case `fail` in some cases), a two-cell, ...". But I also don't mind splitting it up. I think we should choose the version @sebastianpos likes better, than we have a clear vote :D

Reply:
Okay, fair enough :).

Reply:
I flipped a coin. Fabian's version won :-)

Reply:
Fair :)